src/plugins/intel_cpu/src/nodes/reduce.cpp

```diff
@@ -2089,10 +2106,16 @@ void Reduce::initSupportedPrimitiveDescriptors() {
 }
 
 bool Reduce::isExecutable() const {
-    return !isInputTensorAtPortEmpty(REDUCE_DATA);
+    return !isOutputTensorAtPortEmpty(0);
 }
 
 void Reduce::prepareParams() {
+    auto srcMemPtr = getSrcMemoryAtPort(REDUCE_DATA);
+    auto dstMemPtr = getDstMemoryAtPort(0);
+    const auto& src_shape = srcMemPtr->getStaticDims();
+    dst_size = dstMemPtr->getSize();
+    empty_input = shape_size(src_shape) == 0 || srcMemPtr->getSize() == 0;
```

Comment:
Why we need to check both?

Reply:
Applied. Removed getSize() check. Thanks Dmitry!